DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/CourseContent/EditSection.cshtml

```diff
@@ -42,11 +41,13 @@
       </div>
       <input type="hidden" asp-for="CourseName" />
       <input type="hidden" asp-for="SectionName" />
+      <input type="hidden" asp-for="ShowDiagnostic" />
 
       <div class="nhsuk-grid-row  nhsuk-u-margin-bottom-4">
-        <partial name="_EditSectionDiagnostic" model="Model"/>
-
-        <partial name="_EditSectionLearning" model="Model"/>
+        <div @(Model.ShowDiagnostic ? "" : "hidden")>
+          <partial name="_EditSectionDiagnostic" model="Model" />
+        </div>

```

Comment:
I think I'd prefer just wrapping the partial in an if statement like
`if (Model.ShowDiagnostic){ `
`partial`
`}`
Is there some weird layout reason to using the div?

Reply:
Hiding the partial broke model binding, and this was the easiest solution that fixed it. (We.re doing funky binding into lists here)

Reply:
Ah, fair enough then.